wikipedia-dark.user.css

```diff
@@ -2099,8 +2299,8 @@ regexp("https?:\/\/wikiless\.org\/.*$") {
     background-color: var(--gray-2);
   }
   .flow-topic-titlebar, .fieldsetlike .legendtextlike,
-  .oo-ui-tagMultiselectWidget.oo-ui-widget-enabled.oo-ui-tagMultiselectWidget-outlined .oo-ui-tagItemWidget.oo-ui-widget-enabled {
-    background: var(--gray-2);
+  .oo-ui-tagMultiselectWidget.oo-ui-widget-enabled.oo-ui-tagMultiselectWidget-outlined .oo-ui-tagItemWidget.oo-ui-widget-enabled, .thumbimage {
```

Comment:
not sure if thumbimage bg can be anything else other than white, some pages have dark images and then cannot be seen

e..g https://github.com/StylishThemes/Wikipedia-Dark/blob/7636ec668496705476a429264586606d217a72e7/wikipedia-dark.user.css#L4617

Reply:
I have changed those inline styles now and it looks like this: 
![image](https://user-images.githubusercontent.com/86572965/140427516-59a43761-aae6-4bb2-a438-e870298b7807.png)
Pretty neat if you ask me

Its just a matter of changing dropdown menus, the search icon do not appear because its overrided by original code and you said to do not remove anything. Tho I could fix this in pseudo-element rather easly.

Reply:
It didn't even required pseudo element, only added this `form[role=search] input[type=submit]` to transparent background

Reply:
will review tomorrow, now its time for zzz

Reply:
And about `.thumbimage`, most of the images are pure black (#000) not #222 which makes them rather visible, but I will leave this up to you.

Reply:
IIRC this is used in an array of content and its better to leave it untouched.

See https://raw.githubusercontent.com/StylishThemes/Feature-Override-Styles/master/wikipedia-dark-img-styles.user.css which is really where special cases should be handled because it adds more flexibility per user.

However that theme needs to be updated with more specific images as they are found.